### DISCUSSION TITLE
Barcode & QR-Code support

### DISCUSSION THREAD

**Discussion by sohamakl** [typemill/typemill#471] [Ideas]
I'm looking for a solution to include barcodes / qr-codes. Any idea or is a solution already available?

**Comment by trendschau** [comment]
Not familiar with this but I think barcodes are simple images, so you can generate them somewhere and include them as an image into your typemill website. Or did I misunderstand your question?

**Reply by sohamakl** [reply]
I was thinking of a way to generate the QR or bar code directly when writing. 

There is one here. A nice example of successful integration:

https://www.apexofficeprint.com/docs/templates/templates-md/

**Reply by sohamakl** [reply]
Ich dachte hier eher an eine Möglichkeit QR-Code automatisch zu erstellen. So könnte man z.B. einen QR-Code mit Link zu Artikel bei der E-Book-Erstellung mit ausgeben.

vCard als QR-Code wäre auch ein cooles feature.

**Comment by trendschau** [comment]
Thank you, I will check that out for a future release and put this into the task-list.

**Reply by sohamakl** [reply]
Do you already have an idea if and when you can work on the idea?

**Reply by trendschau** [reply]
I checked the APEX docs quickly but this is an highly complicated enterprise software and nothing for Typemill. It is probably possible to integrate some kind of qr-code generation with an individual shortcode plugin for Typemill, but I did not get any similar requests from other users so far so I can only offer to code this as individual software.
